conf.py

```diff
@@ -0,0 +1,71 @@
+# Configuration file for the Sphinx documentation builder.
```

Comment:
Does sphinx need this Python script on project root?

Reply:
Yes, the file has to be in the root as far as i know. The major limitation for sphinx is it cannot include ../xxx.md.